src/Lumi/Components/Modal.purs

```diff
@@ -433,7 +433,10 @@ styles = jss
               , "&:not([data-variant=\"dialog\"])":
                   { "& lumi-modal":
                       { "@media (max-width: 860px)":
-                          { minHeight: "100vh"
+                          { position: "fixed"
+                          , top: "0"
+                          , left: "0"
+                          , height: "100%"
                           , width: "100vw"
                           , maxWidth: "100vw"
```

Comment:
Maybe we should just have this?
```purescript
, top: "0"
, right: "0"
, bottom: "0"
, left: "0"
```
Because the same problem could happen in some mobile browser if it has a sidebar.